openquake/commonlib/source.py

```diff
@@ -536,6 +554,17 @@ def __repr__(self):
 ])
 
 
+def cjoin(strings):
+    """
+    :param strings: an iterable over strings
+    :returns: a comma separated string of lenght <= LENGHT
+    """
+    s = ','.join(strings)
+    if len(s) > LENGTH:
+        raise ValueError('The string %r is over %d characters' % (s, LENGTH))
```

Comment:
I have reduced the error to a warning, since even if happens (which is extremely unlikely) it would break the .rst report, not the computation.